dev/packages/beats/logstash-0.0.1/kibana/search/742e45d0-cbdd-11e7-9852-73e0a9df1bb6-ecs.json

```diff
@@ -21,20 +21,15 @@
                             "alias": null,
                             "disabled": false,
                             "indexRefName": "kibanaSavedObjectMeta.searchSourceJSON.filter[0].meta.index",
-                            "key": "event.module",
+                            "key": "query",
                             "negate": false,
-                            "params": {
-                                "query": "logstash",
-                                "type": "phrase"
-                            },
-                            "type": "phrase",
-                            "value": "logstash"
+                            "type": "custom",
+                            "value": "{\"match_phrase_prefix\":{\"event.dataset\":{\"query\":\"logstash.\"}}}"
                         },
                         "query": {
-                            "match": {
-                                "event.module": {
-                                    "query": "logstash",
-                                    "type": "phrase"
+                            "match_phrase_prefix": {
+                                "event.dataset": {
+                                    "query": "logstash."
```

Comment:
`.` at the end?

Reply:
The field "event.dataset" has value in format "module.dataset" (e.g. nginx.access). The "match_phrase_prefix" function checks the prefix only, so ("nginx.").

Reply:
Got it, makes sense.